packages/cli/src/utils/verifyManifest.ts

```diff
@@ -76,6 +76,28 @@ const endpointCallSchema = object({
 const hostnameRegex =
     /^(([\dA-Za-z]|[\dA-Za-z][\dA-Za-z-]*[\dA-Za-z])\.)*([\dA-Za-z]|[\dA-Za-z][\dA-Za-z-]*[\dA-Za-z])$/;
 
+const ScopeEnum = z.enum([
+    'basic:read',
+    'basic:write',
+    'account:read',
+    'webhook:read',
+    'webhook:write',
+]);
+
+const permissionsSchema = z.object({
+    permissions: z
```

Comment:
Interesting...why not just "scopes"? You think we'll be extending that permissions object to more than scopes?

Reply:
yeah maybe, im was not sure. Checked a bit how other frameworks do it and most have smth inbetween like permissions and there then define it further. 
Could you think of a better alternative?

Reply:
mmmhh not sure... It's not a big deal, just seems extra. Tell me if I miss something but I don't see what we will add to that permissions object here apart from the scopes. Checking quickly what we have for example for Integrations, settings like OAuth clientId, secrets, RedirectUris etc... will be better set in Secure Requests. 🤔

Reply:
yeeh thats true, there is not so much around yet. Maybe what could be possible are some flags like, hasGuidelineAccess or in the theme of the workflows, canAccessWorkflows. In general things that we cannot easily solve through scopes but would like to have an opt-in behaviour.
Maybe more in the networking, but things like whitelisting URLs could also be interesting.

Reply:
Yes, You're right. There could be ways to extend this. Let's keep it as it is.